scss/progressCard.scss

```diff
@@ -0,0 +1,37 @@
+.progress-card__container {
+  display: flex;
+  border-radius: 8px;
+  box-shadow: 0 4px 10px 0 rgba(0, 0, 0, 0.07);
+  background-color: #5438dc;
```

Comment:
Instead to repeat the color everywhere it make more sense to add a CSS variable:
https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties